Mudkip voting bug fix and 23 players bug fix

Xinverse opened the pull request:
- Fixed a bug where multi lynching and abstaining in mudkip breaks the game loop
- Fixed a bug where games cannot run with more than 22 players

Xinverse committed:
Mudkip voting bug fix and 23 players bug fix

Xinverse committed:
Further fix on mudkip lynching

belguawhale commented on bot.py:
```diff
@@ -2983,12 +2983,20 @@ async def reply(message, text, cleanmessage=True):
     await client.send_message(message.channel, message.author.mention + ', ' + str(text))
 
+async def send_long_post(channel, post):
+    if len(post) <= 2000:
```
nit: Please put `2000` in a constant called something like `DISCORD_MAX_MESSAGE_LENGTH`

Xinverse replied:
Done!

belguawhale commented on bot.py:
```diff
@@ -3029,6 +3037,21 @@ async def parse_command(commandname, message, parameters):
         else:
             await log(2, 'User ' + message.author.name + ' (' + message.author.id + ') tried to use command ' + commandname + ' with parameters `' + parameters + '` without permissions!')
 
+async def send_long_log_helper(channel, post, depth):
```
nit: We can use a default argument value here (`depth=0`) so we don't have to specify it when calling this function

Xinverse replied:
Done as well!

belguawhale reviewed (commented):
Nice work, some minor comments.

Xinverse committed:
Message length check and bot status

- Fixed a visual bug where triggering on_start again removes the bot's playing status
- Fixed a bug where longer role tables do not get sent
- Added a constant variable for max discord message length

belguawhale commented on bot.py:
````diff
@@ -3050,7 +3050,7 @@ async def send_long_log_helper(channel, post, depth):
             await client.send_message(channel, "[CONTINUED] " + "```py\n" + post[:max] + "```")
         else:
             await client.send_message(channel, post[:max] + "```")
-        await send_long_log_helper(channel, post[max:], depth + 1)
````
You still need the `depth + 1` here to keep track of the number of continues; you can remove the `0` from https://github.com/belguawhale/Discord-Werewolf/pull/53/files#diff-f289368d1ae6d2d58bec9f799832d932R3070 though

Xinverse replied:
Thanks for catching this mistake! Fixed it in the new commit.

Xinverse committed:
Fixed missing argument in send_long_log_helper()

belguawhale reviewed (approved):
LGTM!
